support-frontend/assets/helpers/productPrice/billingPeriods.js

```diff
@@ -5,14 +5,18 @@ const Monthly: 'Monthly' = 'Monthly';
 const Quarterly: 'Quarterly' = 'Quarterly';
 const SixWeekly: 'SixWeekly' = 'SixWeekly';
 
+// The value for six for six billing period here must match
+// the value in support-models/src/main/scala/com/gu/support/catalog/Product.scala
+const SixForSixBillingPeriod: 'Monthly' = 'Monthly';
```

Comment:
Having the monthly rate called `SixForSixBillingPeriod` feels _really_ confusing to me. I can easily envision getting tripped up on this in the future when making changes to the GW page.

Reply:
This is just a variable so that we can easily switch between using monthly or quarterly as the billing period for six 4 six, once we decide the long term solution it can be removed.

Would it be less confusing if it was:
```javascript
const SixForSixBillingPeriod = Monthly;
```

Reply:
Honestly it's the tying of it to six for six that I think would be a source of confusion- if I was looking at another file referring to this variable, I would assume that `SixForSixBillingPeriod`  meant the initial six week period of the offer rather than 'the recurring rate which you get transferred onto when the offer is up'. I know that effectively it _is_ tied to six for six in terms of churn etc, but I don't know that we ought to be reflecting that directly in the code.

I think something like `ShortRecurringGWBillingPeriod` might be a bit clunkier but more effectively expresses 'this is perhaps Quarterly and perhaps Monthly, but is not Annual and not the six week offer period'.

Reply:
How about `postIntroductorySixForSixBillingPeriod`javascript - it's very verbose but as this is a temporary thing I think it's fine.

Reply:
That sounds good to me!